cwms-data-api/src/test/resources/cwms/cda/api/spk/text_ts_create_reg.json

```diff
@@ -0,0 +1,16 @@
+{
+  "office-id": "SPK",
+  "id": "TsTextTestLoc.Flow.Inst.1Hour.0.raw",
```

Comment:
name vs id?

Reply:
weird that the test didn't catch that.  now I have to double check what is using that resource.